docs/spec_v5_errors.md

```diff
@@ -0,0 +1,173 @@
+---
+id: "spec_v5_errors"
+title: "Twirp Errors Wire Protocol"
+sidebar_label: "Version 5 (Latest)"
+---
+
+This document defines the Twirp wire protocol over HTTP for Errors.
+For more information on the wire protocol see [spec v5](spec_v5.md).
+
+## Twirp Errors
+
+Erros are non-200 JSON responses with the keys:
+
+* **code**: (string) One of the valid Twirp error codes.
+* **msg**: (string) Human-readable message describing the error.
+* **meta**: (optional object) arbitrary additional metadata 
+  describing the error. Keys and values must be strings.
+
+Example:
+```json
+{
+  "code": "internal",
+  "msg": "Something went wrong"
+}
+```
+
+### Content-Type JSON
+
+Twirp error responses are always JSON-encoded, the response Content-Type
+header is always `application/json`, regardless of the request's 
+Content-Type (protbuf requests also respond with json errors).
+This ensures that errors are human-readable and easy to parse by intermediary proxies.
+
+Example Error Response:
+```
+HTTP/1.1 500 Internal Server Error
+Content-Type: application/json
+Content-Length: 48
+
+{"code":"internal","msg":"Something went wrong"}
+```
+
+### Twirp Error Codes
+
+The possible values for the Error "code" are intentionaly similar to
+[gRPC status codes](https://godoc.org/google.golang.org/grpc/codes):
+	
+* **canceled**: The operation was cancelled (typically by the caller).
+* **unknown**: Unknown error. For example when handling errors raised by APIs 
```

Comment:
Rephrase this to complete sentences: `"An unknown error occurred. For example, this can be used when handling errors raised by APIs that do not return any error information."`